web/src/hashchange.js

```diff
@@ -258,7 +258,9 @@ function do_hashchange_overlay(old_hash) {
     // the new overlay.
     if (coming_from_overlay && base === old_base) {
         if (base === "streams") {
-            stream_settings_ui.change_state(section);
+            // e.g. #streams/29/social/subscribers
```

Comment:
Hmm, `/29-social/` is the convention we use for streams in `#narrow` URLs, rather than `/29/social/. It's probably not worth much time to change this now with other priorities, but it might be worth opening an issue for it.